css/style.css

```diff
@@ -0,0 +1,187 @@
+* {
+    padding: 0;
+    margin: 0;
+}
```

Comment:
- [ ] You are attempting here to reset your browser defined CSS. To do this consider going to this [link](https://github.com/necolas/normalize.css/) and downloading the Normalize.css file and including it in your project.